dashboard/test/models/user_test.rb

```diff
@@ -1685,6 +1685,30 @@ def track_progress(user_id, script_level, result, pairings: nil)
     create :teacher
   end
 
+  test 'admin_changed? equates nil and false' do
+    user = create :user
+    admin = create :admin
+
+    # no change to admin, false
+    refute user.admin_changed?
+    refute admin.admin_changed?
+
+    user.admin = false
+    refute user.admin_changed?
+
+    user.admin = nil
+    refute user.admin_changed?
+
+    user.admin = true
+    assert user.admin_changed?
+
+    admin.admin = nil
+    assert admin.admin_changed?
+
+    admin.admin = false
+    assert admin.admin_changed?
```

Comment:
How so? Do you mean it just isn't readable? Or do you think it's missing coverage? We could do a matrix of all combinations of: nil, false, true